compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/window/Dialog.skiko.kt

```diff
@@ -208,32 +208,25 @@ private fun DialogLayout(
             containerSize = containerSize,
             platformInsets = platformInsets
         )
-        PlatformInsetsConfig.excludeInsets(
```

Comment:
Looks suspecious. Does `usePlatformInsets` flag work correctly now?
Verify `WindowInsets` values inside dialog before/after this change

Reply:
This behavior should be restored. Removing the exclude logic was not correct.